frontend/src/queries/schema.ts

```diff
@@ -582,15 +569,28 @@ export interface WebTopClicksQueryResponse extends QueryResponse {
     columns?: unknown[]
 }
 
-export interface WebTopPagesQuery extends WebAnalyticsQueryBase {
-    kind: NodeKind.WebTopPagesQuery
+export enum WebStatsBreakdown {
```

Comment:
I used an enum here because some properties will take special handling on the query side, so I can't just pass a property name